code/modules/plumbing/plumbers/synthesizer.dm

```diff
@@ -61,8 +63,25 @@
 		return
 	if(reagents.total_volume >= amount*delta_time*0.5) //otherwise we get leftovers, and we need this to be precise
 		return
+	if(volume_left < amount || !volume_left)
```

Comment:
0 is always smaller than amount

Reply:
``volume_left`` isn't 0 unless the synth is empty, ``amount`` is how many units get dispensed per cycle

Reply:
My point is that you will never fail second condition

Reply:
Second condition fails unless ``volume_left`` is null? Or am I missing something here

Reply:
Besides ``volume_left`` is increased by 400 when an RCD ammo is used on the synth, failing both checks until ``volume_left`` is below ``amount``

Reply:
"fixed" even though it was working perfectly well during testing.